### PR TITLE
Metrix fake

### PR DESCRIPTION
**Pull request by dabeat**
#### :tophat: Problem
This module is needed in all services for testing metrix.

#### :pushpin: Solution
Make it common.

#### :ghost: GIF
giphy

### CODE REVIEW

File: lib/test_helper/metrix_fake.ex [hunk 1]
```diff
@@ -0,0 +1,15 @@
+defmodule ElixirTools.TestHelper.MetrixFake do
```
Comment (Kitton) [inline lib/test_helper/metrix_fake.ex]: tests are missing

**Comment by dabeat** [comment]
In another PR @fartek

File: lib/test_helper/metrix_fake.ex (hunk 1 above)
Comment (fartek) [reply lib/test_helper/metrix_fake.ex]: Wait, we test fakes?

Comment (Kitton) [reply lib/test_helper/metrix_fake.ex]: yep, why not

Comment (fartek) [reply lib/test_helper/metrix_fake.ex]: Well we could also test tests but it would be a bit of a waste of time, right? :D

**Pull request merged**